ci/deploy-rpm.sh

```diff
@@ -1,18 +1,24 @@
-#!/bin/sh
+#!/bin/bash
 
-RPM_EL6=$(find dist/ -type f -name "*64bit.rpm" -printf "%f\n" | head -n1 | sed -e 's/_/-/g' -e 's/-Linux/.el6/' -e 's/-64bit/.x86_64/')
-RPM_EL7=$(find dist/ -type f -name "*64bit.rpm" -printf "%f\n" | head -n1 | sed -e 's/_/-/g' -e 's/-Linux/.el7/' -e 's/-64bit/.x86_64/')
+function create_rpm_repo () {
+        version=$1
 
-cd trivy-repo
-mkdir -p rpm/releases/6/x86_64
-mkdir -p rpm/releases/7/x86_64
+        RPM_EL=$(find ../dist/ -type f -name "*64bit.rpm" -printf "%f\n" | head -n1 | sed -e "s/_/-/g" -e "s/-Linux/.el$version/" -e "s/-64bit/.x86_64/")
+        echo $RPM_EL
+
+        mkdir -p rpm/releases/${version}/x86_64
```

Comment:
NIT: `rpm/releases/${version}/x86_64` occurs 3 times in the script so you could probably assign it to a variable, e.g. `rpms_path`

Reply:
Thank you for the review! I fixed it.